src/snarky.d.ts

```diff
@@ -28,6 +28,8 @@ export { Test };
  * `Provable<T>` is the general circuit type interface. It describes how a type `T` is made up of field elements and auxiliary (non-field element) data.
  *
  * You will find this as the required input type in a few places in snarkyjs. One convenient way to create a `Provable<T>` is using `Struct`.
+ * 
+ * The properties and methods on the `Provable` type exist in all base snarkyjs types as well (aka. {@link Field}, {@link Bool}, etc.). In most of the cases, a ZK app developer does not need these functions to create dApps.
```

Comment:
```suggestion
 * The properties and methods on the `Provable` type exist in all base SnarkyJS types as well (aka. {@link Field}, {@link Bool}, etc.). In most cases, a ZK app developer does not need these functions to create Dapps.
```

Reply:
I really like this description, thank you @yunus433 

See the word list in the docs style guide for spelling and capitalization guidance. We have inconsistencies in our docs and are working to establish consistency. Dapps is one example of how we have landed on the Dapp spelling even when that spelling might not be the most common